pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/assignment/segment/OfflineSegmentAssignment.java

```diff
@@ -72,10 +81,12 @@
   private String _offlineTableName;
   private int _replication;
   private String _partitionColumn;
+  private TableConfig _tableConfig;
```

Comment:
(OCD) move this to line 81

Reply:
even better, removed it because we no longer are checking `shouldRelocateToTiers(tableConfig)` in this file